project/BuildHelper.scala

```diff
@@ -294,7 +294,7 @@ object BuildHelper {
   )
 
   def nativeSettings = Seq(
-    Test / skip := true,
+    Test / test / skip := true,
```

Comment:
Does this have any effect? I don't see where `skip` is looked up within `test` in sbt.

Reply:
Ha! Maybe it actually doesn't have any effect, but now at least `Test` isn't disabled entirely, so (sbt-)scalafix can work with it.
But how can we prevent `Test / test` from running the tests then? `Test / test := {}`?

Reply:
> Ha! Maybe it actually doesn't have any effect, but now at least Test isn't disabled entirely, so (sbt-)scalafix can work with it.

:+1: it also makes sure that shared test sources build for all platforms, which wasn't the case despite the explicit reference in https://github.com/zio/zio-prelude/blob/b6ca2944eb5703561f27bef187b96cdfa9c8039c/build.sbt#L30-L33 

>  But how can we prevent `Test / test` from running the tests then? `Test / test := {}`?

I believe that would work, yes.

I assume CI skips it because of the usage of the command below, but a `test` at the root probably runs the native tests now (and wasn't before because the test framework wouldn't find anything compiled).